lightning/src/ln/channelmanager.rs

```diff
@@ -7414,6 +7477,28 @@ where
 		BlindedPath::one_hop_for_message(self.get_our_node_id(), entropy_source, secp_ctx).unwrap()
 	}
 
+	/// Creates a one-hop blinded path with [`ChannelManager::get_our_node_id`] as the introduction
+	/// node.
+	fn create_one_hop_blinded_payment_path(
+		&self, payment_secret: PaymentSecret
+	) -> (BlindedPayInfo, BlindedPath) {
+		let entropy_source = self.entropy_source.deref();
+		let secp_ctx = &self.secp_ctx;
+
+		let payee_node_id = self.get_our_node_id();
+		let payee_tlvs = ReceiveTlvs {
+			payment_secret,
+			payment_constraints: PaymentConstraints {
+				max_cltv_expiry: LATENCY_GRACE_PERIOD_BLOCKS,
```

Comment:
This should be set to an absolute expiry

Reply:
Oh, I think I see why you thought this was correct... `PaymentConstraints::max_cltv_expiry` docs are wrong.

Reply:
Hmm, the spec indicates that `max_cltv_expiry` is used to restrict the lifetime of blinded paths (if you search `max_cltv_expiry` in BOLT 4, hard to link to), and is intended to be compared to `msg.cltv_expiry` (which is absolute). Don't recall the offline discussion unfortunately :(